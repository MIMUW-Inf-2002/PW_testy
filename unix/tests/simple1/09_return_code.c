#include <stdio.h>
#include <stdlib.h>
#include <unistd.h>

int main(int argc, char *argv[])
{
    // Check if a command line argument was provided
    if (argc < 2) {
        usleep(300000);
        return 0;
    }

    // Get the return code from the command line argument
    int return_code = atoi(argv[1]);
    printf("%d\n", return_code);

    // Return the specified return code
    return return_code;
}
