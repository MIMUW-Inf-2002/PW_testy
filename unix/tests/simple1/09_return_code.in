run ./09_return_code 1
run ./09_return_code 2
run ./09_return_code 3
sleep 100
out 1
