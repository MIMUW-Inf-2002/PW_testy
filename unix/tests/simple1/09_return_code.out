Task 0 started: pid 66039.
Task 0 ended: status 2.
Task 0 stdout: ''.
